stix_shifter_modules/arcsight/stix_translation/transformers.py

```diff
@@ -14,8 +14,9 @@ def transform(registry_key):
                                   "HKPD": "HKEY_PERFORMANCE_DATA", "HKU": "HKEY_USERS", "HKDD": "HKEY_DYN_DATA"}
         try:
             root_key = registry_key.split("\\")
-            map_root_key = stix_root_keys_mapping[root_key[0]]
-            root_key[0] = map_root_key
+            if splited[0] in stix_root_keys_mapping:
```

Comment:
`splited` is never defined. It's causing the builds to fail. Needs to be set to `root_key` in this case.